plugins/cpu/fu-cpu-helper-cet.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
```

Comment:
Presumably `H.J. Lu <hjl.tools@gmail.com>` should be in this?

Reply:
Yes.  LGTM.

Reply:
Agree, thanks.